languages/csharp/src/main/java/de/jplag/csharp/CSharpLanguage.java

```diff
@@ -25,7 +25,7 @@ public String[] suffixes() {
 
     @Override
     public String getName() {
-        return NAME;
+        return "C#";
```

Comment:
With this change, you produce inconsistencies with the `name` constant. You should probably just adapt the constant value instead.